zebra-chain/src/parameters/network/subsidy.rs

```diff
@@ -233,6 +318,13 @@ impl ParameterSubsidy for Network {
         }
     }
 }
+
```

Comment:
```suggestion

// TODO: Try to unify in one implementation with the above `impl ParameterSubsidy for &Network`
```

Reply:
I figured out how to get rid of it, fixed in https://github.com/ZcashFoundation/zebra/pull/8718/commits/852d63438288cd433e6c2cbd88c4326ba4765e56.